requirements.txt

```diff
@@ -1 +1 @@
-torch >= 0.0.0.dev0
+torch >= 1.12.0.dev0
```

Comment:
Are we requiring 1.12 or master? `>= 1.12` implies that `1.12` will work, but if we depend on master that wouldn't be true

Reply:
good question, but this check is better than nothing :) PyPPy for sure doesn't work with < 1.12